util/randutil/rand.go

```diff
@@ -47,6 +47,11 @@ func RandIntInRange(r *rand.Rand, min, max int) int {
 	return min + r.Intn(max-min)
 }
 
+// RandInt64InRange returns a value in [min, max)
+func RandInt64InRange(r *rand.Rand, min, max int64) int64 {
```

Comment:
I'm not sure this is worth adding to `util/randutil` given that it is only used in your simulation. And you don't even need this generality in your simulation (yet) since you're always calling this function with `min == 0`.